src/GuiBase/Timeline/Configurations.h

```diff
@@ -0,0 +1,61 @@
+#ifndef RADIUMENGINE_TIMELINE_CONFIG_HPP_
+#define RADIUMENGINE_TIMELINE_CONFIG_HPP_
+
+// fix timeline's width directly in animtimeline.ui
```

Comment:
Is the timeline width's fix ? At runtime, user don't have accezs to the .ui. 
It is not a good idea to have fixed values for ui elements. You have no information on the display capabilities on the user workstation and a fix value can prevent to use the application. (e.g. try to use main-app on WXGA monitor and you will understand why fixed values are not so good).

Reply:
remnant that should have been gone